backend/dal/Services/Concrete/UserService.cs

```diff
@@ -108,6 +110,21 @@ public AccessRequest AddAccessRequest(AccessRequest request)
             this.Context.CommitTransaction();
             return request;
         }
+
+        /// <summary>
+        /// Get an array of agency IDs for the specified 'userId'.
+        /// This only returns the first two layers (direct parents, their immediate children).
```

Comment:
why only the two layers?

Reply:
This is a design decision.  I design the db to support more for future proofing, but the business requirements only demand two layers ATM.  The performance implications for more are exponential.